src/containers/announcements/ducks/types.ts

```diff
@@ -0,0 +1,23 @@
+import { AsyncRequest } from '../../../utils/asyncRequest';
+import { C4CState } from '../../../store';
+import { ThunkAction } from 'redux-thunk';
+import { AnnouncementsActions } from './actions';
+import { ApiExtraArgs } from '../../../api/publicApiClient';
+
+export interface AnnouncementsReducerState {
+    readonly announcements: AsyncRequest<AnnouncementProps[], any>;
+}
+
+export type AnnouncementsThunkAction<R> = ThunkAction<
+    R,
+    C4CState,
+    ApiExtraArgs,
+    AnnouncementsActions
+>;
+
+export interface AnnouncementProps {
```

Comment:
```suggestion
export interface Announcement {
```
"Props" should only be used to define the actual prop interfaces for components. You can use an IDE to refactor this easily.

Reply:
It is the props for the AnnouncementCard